optimum/intel/openvino/modeling_decoder.py

```diff
@@ -419,7 +420,7 @@ def prepare_inputs(
                             shape[2] = 0
                         else:
                             shape[1] = 0
-                    inputs[input_name] = Tensor(model_inputs.get_element_type(), shape.get_shape())
+                    inputs[input_name] = np.empty([dim.get_length() for dim in shape], dtype=dtype)
```

Comment:
Looks like instances of `openvino.runtime.Tensor` cannot be pickled which is needed for the fix added in https://github.com/huggingface/optimum-intel/pull/632 for the `InferRequestWrapper`

Reply:
I hope it will not break the tracing of models. cc'ed @eaidova, @slyalin.